package.json

```diff
@@ -32,7 +32,7 @@
     "prettier-check": "prettier --check \"src/**/*.{ts,js,json,yml}\" --ignore-path .gitignore || (echo \"\nYou can fix this by running ==> yarn prettier-write <==\n\" && false)",
     "prettier-write": "prettier --write \"src/**/*.{ts,js,json,yml}\" --ignore-path .gitignore",
     "test": "jest",
-    "test:debug": "node --inspect-brk `which jest`",
+    "test:debug": "node --inspect-brk `which jest` --runInBand",
```

Comment:
unrelated change: `--inspect-brk` does nothing if you're running tests in parallel